src/components/DepositWidget/useRowActions.tsx

```diff
@@ -55,8 +58,13 @@ export const useRowActions = (params: Params): Result => {
   }
 
   async function enableToken(tokenAddress: string): Promise<void> {
-    const { symbol } = getToken('address', tokenAddress, balances)
+    const { symbol } = getToken('address', tokenAddress, balances) as Required<TokenBalanceDetails>
```

Comment:
ubernit: I think it's good when we add a `Required` to include a comment on why we know it contains the properties. Also, I think in that case, it was used as a shortcut for saying that it has a symbol for sure.

Important: Why we assume here that it has a symbol? I think this might not be true.

I think I understand why you did it, but in this case, the symbol is used only for giving some feedback to the user. In this case I would do:

```js
const { symbol } = getToken('address', tokenAddress, balances)
const label = symbol || tokenAddress
toast.success(`The token ${label} has been enabled for trading`)
```

Also, if we see this pattern repeated 10 times in the app, we might consider to add this `label` param as mandatory, and we compute it when we load the tokens. This way, the children don't need to decide.

Reply:
On that note, I created this util function awhile ago for exactly this use case: https://github.com/gnosis/dex-react/blob/2ae24e9dab20646d583796bead91664fa4cf849a/src/utils/format.ts#L133